src/app/pages/public/LifePage/index.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import BasicLayout from '../../../layouts/BasicLayout';
+import { useGetLifeLazyQuery } from '../../../api';
+import NotFoundResult from '../NotFoundPage';
+import LoadingElement from '../../../components/LoadingElement';
+import { Descriptions, Tag } from 'antd';
+
+const LifePage = () => {
+    const { id } = useParams();
+    console.log('id: ', id);
+    const [fetch, { data, loading }] = useGetLifeLazyQuery();
+
+    useEffect(() => {
+        fetch({ variables: { id } });
+    }, []);
+
+    if (loading) return <LoadingElement />;
+    if (!data) return <NotFoundResult />;
+    console.log('data: ', data);
+    return (
+        <BasicLayout>
+            {data && (
+                <Descriptions title="Life Info">
+                    <Descriptions.Item label="FullName">{data?.getLife?.fullName}</Descriptions.Item>
+                    <Descriptions.Item label="Birthday">{data?.getLife?.birthday}</Descriptions.Item>
+                    <Descriptions.Item label="Title">{data?.getLife?.title}</Descriptions.Item>
+                    <Descriptions.Item label="Description">{data?.getLife?.description}</Descriptions.Item>
```

Comment:
`data` is not `null` or `undefined` since your ternary already checked that out, you should better do something like this ;

```ts
const life = data?.getLife;

if (!life) {
  return (/* ... */);
}

// here you can call life.something without worry

return (/* ... */);
```